screens/wishlist/Wishlist.css

```diff
@@ -0,0 +1,59 @@
+.product-section-container {
+  width: 90%;
+}
+
+.grid-col-4 {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 10px;
+}
+
+.grid-item {
+  margin: 0.8rem;
+}
+
+.grid-item-img {
+  position: relative;
+}
+
+.grid-item-icon {
+  position: absolute;
+  top: 1rem;
+  right: 1.2rem;
+  cursor: pointer;
+}
+
+.grid-item-icon i {
+  font-size: 1.5rem;
+  color: var(--error-color);
+}
+
+@media screen and (max-width: 1250px) {
+  /* products */
+  .grid-col-4 {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 10px;
+  }
+}
+
+@media screen and (max-width: 820px) {
+  .grid-col-4 {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 10px;
+  }
+}
+
+@media screen and (max-width: 600px) {
+  .grid-col-4 {
+    display: grid;
+    grid-template-columns: 1fr;
+    gap: 10px;
+  }
```

Comment:
There seems to be some redundant code in these media queries can we remove the repeated parts i.e (display:grid , gap:10px) from screen sizes 820px and 600px and 1024px because it has already been mentioned in line 5 to 9 the same styles would get applied u only have to change the grid-template-columns part for responsive design